### DISCUSSION TITLE
Can ProxyMan listen on multiple interfaces at the same time?

### DISCUSSION THREAD

**Discussion by igorwojda** [ProxymanApp/Proxyman#2149] [Q&A] [answered]
Can ProxyMan listen on multiple interfaces (WiFi and LAN)?

**Comment by NghiaTranUIT** [comment, marked as answer]
when Proxyman is active, it tries to override all network interfaces on your Mac, including the LAN and WiFi, Sharing Wi-Fi from iPhone if needed 👍

**Reply by igorwojda** [reply]
In that case it may be good improvement to present all ip addresses at the top of the UI (not only one).

Also icons representing connection type (WiFi/Lan) would be a good addition.
